addons/l10n_lv/data/account_account_tag_data.xml

```diff
@@ -0,0 +1,1424 @@
+    <field name="applicability">accounts</field>
+    <field name="color" eval="8"/>
+  </record>
+
+<record id="account_account_tag_pza2_210" model="account.account.tag">
+    <field name="name">PZA2.210. Procentu maksājumi un tamlīdzīgas izmaksas</field>
+    <field name="applicability">accounts</field>
+    <field name="color" eval="8"/>
+  </record>
+
+<record id="account_account_tag_pza2_220" model="account.account.tag">
+    <field name="name">PZA2.220. a) radniecīgajām sabiedrībām</field>
+    <field name="applicability">accounts</field>
+    <field name="color" eval="8"/>
+  </record>
+
+<record id="account_account_tag_pza2_230" model="account.account.tag">
+    <field name="name">PZA2.230. b) citām personām</field>
+    <field name="applicability">accounts</field>
+    <field name="color" eval="8"/>
+  </record>
+
+<record id="account_account_tag_pza2_240" model="account.account.tag">
+    <field name="name">PZA2.240. Peļņa vai zaudējumi pirms uzņēmumu ienākuma nodokļa</field>
+    <field name="applicability">accounts</field>
+    <field name="color" eval="8"/>
+  </record>
+
+<record id="account_account_tag_pza2_250" model="account.account.tag">
+    <field name="name">PZA2.250. Uzņēmumu ienākuma nodoklis par pārskata gadu</field>
+    <field name="applicability">accounts</field>
+    <field name="color" eval="8"/>
+  </record>
+
+<record id="account_account_tag_pza2_260" model="account.account.tag">
+    <field name="name">PZA2.260. Peļņa vai zaudējumi pēc uzņēmumu ienākuma nodokļa aprēķināšanas</field>
+    <field name="applicability">accounts</field>
+    <field name="color" eval="8"/>
+  </record>
+
+<record id="account_account_tag_pza2_270" model="account.account.tag">
+    <field name="name">PZA2.270. Ieņēmumi vai izmaksas no atliktā nodokļa aktīvu vai saistību atlikumu izmaiņām</field>
+    <field name="applicability">accounts</field>
+    <field name="color" eval="8"/>
+  </record>
+
+<record id="account_account_tag_pza2_280" model="account.account.tag">
+    <field name="name">PZA2.280. Ārkārtas dividendes</field>
+    <field name="applicability">accounts</field>
+    <field name="color" eval="8"/>
+  </record>
+
+<record id="account_account_tag_pza2_290" model="account.account.tag">
+    <field name="name">PZA2.290. Pārskata gada peļņa vai zaudējumi.</field>
+    <field name="applicability">accounts</field>
+    <field name="color" eval="8"/>
+  </record>
+
+
+    <!-- Tags for taxes -->
+
+     <!-- VAT report 1.1. types -->
+
+     <!-- ieģāde ar PVN 21/12/5% iegādi, Rinda 62 + PVN 1.1. -->
+    <record id="VAT_11SA_tag" model="account.account.tag">
+    <field name="name">Summa 1.1A</field>
+    <field name="applicability">taxes</field>
+    <field name="country_id" ref="base.lv"/>
+    </record>
+
+     <record id="VAT_11PA_tag" model="account.account.tag">
+    <field name="name">PVN 1.1A</field>
+    <field name="applicability">taxes</field>
+    <field name="country_id" ref="base.lv"/>
+    </record>
```

Comment:
please re-indent this file

Reply:
what about directly linking with a tax report?